src/gearbox/config.py

```diff
@@ -133,4 +124,7 @@ def __init__(self, value):
 
 GEARBOX_MIDDLEWARE_PUBLIC_KEY_Path = config("GEARBOX_MIDDLEWARE_PUBLIC_KEY_PATH", default='src/gearbox/keys/jwt_public_key.pem')
 
-GEARBOX_KEY_CONFIG = {}
\ No newline at end of file
+GEARBOX_KEY_CONFIG = {}
+
+# DOCCANO placeholder for new criterion
+DOCCANO_PLACEHOLDER = config("DOCCANO_PLACEHOLDER", default="new_variable")
```

Comment:
Could we have a more specific name in case we'll need to have other values from Doccano in the config? Something like DOCCANO_MISSING_VALUE_PLACEHOLDER